docs/test/live-unit-testing-start.md

```diff
@@ -15,7 +15,7 @@ ms.workload:
 ---
 # Get started with Live Unit Testing in Visual Studio
 
-When you enable Live Unit Testing in a Visual Studio solution, Live Unit Testing visually depicts your test coverage and the status of your tests. It also dynamically executes tests whenever you modify your code. It provides immediate notification when changes have broken your code and indicates areas for which additional tests are needed.
+When you enable Live Unit Testing in a Visual Studio solution, Live Unit Testing visually depicts your test coverage and the status of your tests. It also dynamically executes tests whenever you modify your code and immediately notifies you when your changes cause tests to fail.
```

Comment:
It also graphically depicts code coverage and code paths that are not covered by existing tests, which was the intent of the end of the original last sentence. I think that continues to be important to note.

Reply:
Ok, I'll think of a different way to phrase it. As it is now, it seems kind of tacked on.

Reply:
Yes, I agree, @jpreese.